spec/rubocop/cop/thread_safety/mutable_class_instance_variable_spec.rb

```diff
@@ -282,7 +282,7 @@ def some_method
             it_behaves_like 'mutable objects', %("\#{a}")
           end
 
-          context 'when the frozen_string_literal comment is true' do
+          context 'when the frozen_string_literal comment is true', unsupported_on: :prism do
```

Comment:
This will make RSpec skip it on prism and let us know when it starts working because RSpec enforces pending specs must fail.

https://rspec.info/features/3-12/rspec-core/pending-and-skipped-examples/

```suggestion
          context 'when the frozen_string_literal comment is true' do
            pending 'Unsupported on prism' if ENV['PARSER_ENGINE'] == 'parser_prism'
```

We could consider extracting a helper:

```ruby
def prism?
  ENV['PARSER_ENGINE'] == 'parser_prism'
end
```

Reply:
Thanks, I didn't know about this feature. However, is this case it's not a prism-related issue, this spec only passes with Ruby 2.7 parser (dynamic strings are not frozen by default since 3.0), and while all the specs run with 2.7 parser, setting `PARSER_ENGINE` to `parser_prism` [automatically promotes](https://github.com/rubocop/rubocop/blob/51f29dcc173636f18cf0507613ca0d500d656cbe/lib/rubocop/rspec/cop_helper.rb#L11) the parser to 3.3. So this spec will never pass on prism.

Reply:
I added specs for default parser that run at Ruby 3.0 (it now covers [this](https://github.com/rubocop/rubocop-thread_safety/blob/master/lib/rubocop/cop/thread_safety/mutable_class_instance_variable.rb#L130) line).